Makefile

```diff
@@ -6,7 +6,7 @@ UNAME_S := $(shell uname -s)
 
 # If you're using docker and docker-compose, you can use this Makefile to run
 # commands in your docker images by setting the DOCKER_PREFIX environment variable
-# to: DOCKER_PREFIX="docker-compose run --rm web"
+# to: DOCKER_PREFIX="docker-compose run --rm web --"
```

Comment:
The `DOCKER_PREFIX` trick is to allow running `make` outside of the container, and have it run all of the individual commands inside it. The problem that Andreas was trying to fix is that, with the current `start-docker.sh` script, that command needs a `--` at the end of it (which should be fixed in `start-docker.sh` instead). The problem that I'm talking about is that it makes more sense to just run `docker-compose run web --rm make foo` than to set `DOCKER_PREFIX` to `docker-compose run web --rm` and then run `make foo`, because the latter results in 5 containers being constructed and torn down for a make target with 5 commands, without actually significantly simplifying anything for anyone.